examples/02_decoding/plot_haxby_full_analysis.py

```diff
@@ -44,49 +45,53 @@
 # extract tags indicating to which acquisition run a tag belongs
 session_labels = labels["chunks"][task_mask]
 
+# apply the task_mask to  fMRI data (func_filename)
+from nilearn.image import index_img
+task_data = index_img(func_filename, task_mask)
 
 ##########################################################################
-# Then we use scikit-learn for decoding on the different masks
-# -------------------------------------------------------------
-# The classifier: a support vector classifier
-from sklearn.svm import SVC
-classifier = SVC(C=1., kernel="linear")
+# Decoding on the different masks
+# --------------------------------
 
-# A classifier to set the chance level
-from sklearn.dummy import DummyClassifier
-dummy_classifier = DummyClassifier()
```

Comment:
For the ":class:" markup to be useful, you need to insert a comment block by avoiding the blank line above.